validator_test.go

```diff
@@ -301,14 +302,21 @@ func TestValidatorSubDomains(t *testing.T) {
 		},
 	}
 
-	dir, err := ioutil.TempDir("", "emailstest")
-	if err != nil {
-		log.Fatal(err)
-	}
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
 			g := NewWithT(t)
 
+			// Make a TempFile prefix somehow based on the testcase name
```

Comment:
Lol - This comment was just for your benefit in the sample -- not intended to go into the codebase 🤣